docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+# Example file for a Drupal-based setup.
```

Comment:
Måske omdøb filen til docker-compose.example.yml :)

Reply:
@rasben Good point, men hele repo'et er et eksempel (proref = "Project reference"), så tænker det er lidt omsonst. Jeg ville gerne gøre det muligt at køre projektet up for realz uden videre, efter man har klonet det.